Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Linker/LinkerPleaseInclude.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Specialized;
+using System.Windows.Input;
+using Android.App;
+using Android.Views;
+using Android.Widget;
+
+namespace MvvmSeed.Android.Linker
+{
+    /// <summary>
+    /// This file is required to avoid android Linker wiping references to code not detected during static analysis.
+    /// (We could disable linking, but I rather not, no linking means x3 or x4 size for builds)
+    /// </summary>
+    public class LinkerPleaseInclude
```

Comment:
What if we specified manually the bindings? It would also give us finer control over the bindings. Yes, we would end up with a huge file that has access to everything, which is a downside, but it prevents us from having to do any of this and it also increases visibility on the configuration of the bindings.